kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzConfig.kt

```diff
@@ -15,7 +15,8 @@ import kotlin.time.Duration.Companion.seconds
  *
  * @param fuzzEngine - name of engine to be used. Default: "jazzer"
  * @param hooks - apply fuzzing instrumentation. Default: true
- * @param keepGoing - how many bugs to discover before finishing fuzzing. Default: 1
+ * @param keepGoing - Maximum number of new and unique bugs to be discovered before finishing fuzzing. Duplicates of both old and new will not be counted.
```

Comment:
Not to be fixed right now, but just a thought for the future: do we need `keepGoing` to be an actual parameter? In what scenario will user want to use it?

Reply:
I think that we have it mostly because jazzer had it. So, why not add it? But I agree that users in most scenarios will set it to a 0 or some large value to just fuzz as long as possible